client/landing/stepper/declarative-flow/internals/steps-repository/plans/plans-wrapper.tsx

```diff
@@ -75,7 +75,8 @@ const PlansWrapper: React.FC< Props > = ( props ) => {
 	}, [] );
```

Comment:
**Note:**

Amendments to this file allow us to add storage add-ons to the shopping cart in stepper onboarding flows